Makefile

```diff
@@ -21,15 +21,15 @@ check-commands:
 	done
 
 setup: COMMANDS=rustup pre-commit
-setup: check-commands 
+setup: check-commands
 	./scripts/setup.sh
 
 # ------------------------------------------------------------
 #  Development
 # ------------------------------------------------------------
-
-dev:
-	cargo run
+# Starts fuel-core-nats service
+start.fuel-core-nats:
```

Comment:
Ended up with `start` to allow the use of `stop` DSL in the script for stopping `docker-compose` services